commercetools-sdk-base/src/main/java/io/sphere/sdk/queries/PagedResult.java

```diff
@@ -50,6 +50,32 @@
      }
 
     /**
+     * Calculates the page number of the result.
+     *
+     * @return the page number of the result, 0 if the page size is 0, meaning "{@code getCount()}" returns 0.
+     */
+    default Long getPage() {
+
+        if(getCount() == 0)
+            return 0L;
+
+        return getOffset() / getCount();
+    }
+
+    /**
+     * Calculates the total number of pages matching the request.
+     *
+     * @return the total number of pages , 0 if the page size is 0, meaning "{@code getCount()}" returns 0.
+     */
+    default Long getPagesCount() {
+
+        if(getCount() == 0)
+            return 0L;
+
+        return   getTotal() / getCount();
```

Comment:
My implementation looks a bit different from it:
```
        if (pagedResult.isLast()) {
            return currentPage;
        } else if (pagedResult.getCount() > 0) {
            final Double totalPages = Math.ceil(pagedResult.getTotal() / pagedResult.getCount());
            return totalPages.longValue();
        } else {
            return 0;
        }
```
But let's check after you write more tests.